projects/sonar/src/app/record/document/aggregation-filter.ts

```diff
@@ -35,11 +35,17 @@ export class AggregationFilter {
     Object.keys(aggregations).forEach(aggregation => {
       // Translate values for document type
       if (aggregation === 'document_type') {
-        aggregations[aggregation].buckets.forEach((bucket) => {
+        aggregations[aggregation].buckets.forEach((bucket: any) => {
           bucket.name = this.translateService.instant('document_type_' + bucket.key);
         });
       }
 
+      if (aggregation === 'status') {
+        aggregations[aggregation].buckets.forEach((bucket: any) => {
+          bucket.name = this.translateService.instant('deposit_status_' + bucket.key);
```

Comment:
how do you extract this for the translations?

Reply:
In manual translations: https://github.com/rero/sonar-ui/blob/dev/projects/sonar/src/manual_translations.ts#L151